### ISSUE TITLE
Websockets output change

### ISSUE THREAD

**Issue by Robert-j7** [JuliaWeb/HTTP.jl#1047]
Does the `WebSockets.open` returns an abstract type? Is it documented?

**Comment by Robert-j7** [comment]
Yeah, I was wondering about `HTTP.WebSockets.WebSocket` which returns the fields defined below. We would have gotten only the `:readbuffer` when you are receiving data from a websockets as strings. 

`(:id, :io, :request, :response, :maxframesize, :maxfragmentation, :client, :re
adbuffer, :writebuffer, :readclosed, :writeclosed)                            
`

**Comment by quinnj** [comment]
I still don't understand the question/issue; can you rephrase perhaps?

**Comment by Robert-j7** [comment]
This is weird! For debugging I just use `println` to see what's happening like this:

```
WebSockets.open("wss://fstream.binance.com/stream?streams=btcusdt@depth") do ws
        for msg in ws
               println(msg)
        end
end
```
And it would print alright. After that I push the data I want to a database, now two days ago I just send the incoming data to an empty array like this:

```
WebSockets.open("wss://fstream.binance.com/stream?streams=btcusdt@depth") do ws
        for msg in ws
               push!(array, msg)
        end
end
```
when I did this it would just send `:id :io :maxframesize...' but not the data itself that should be inside `:readbuffer`. So I just put `msg.readbuffer` instead of `msg` and it worked. So I thought there were changes to the `WebSockets` output type. I tried today again, everything is back to normal, no need for the `msg.readbuffer`, it just sends the data into array!

**Comment by Robert-j7** [comment]
To add, the issue I encountered was 1.9.2, and the issue was gone in 1.9.4, both Julia 1.9. Unfortunately I didn't log the problematic output. If you think it is not that important, I will just close this issue.

**Comment by quinnj** [comment]
Yeah, there was a bug in the 1.9.2/1.9.3 releases that was fixed in the 1.9.4 release.
